app/models/comfy/cms/page.rb

```diff
@@ -3,14 +3,28 @@
 class Comfy::Cms::Page < ActiveRecord::Base
   scope :most_popular, -> (number_of_items) { unscoped.order('page_views desc').limit(number_of_items) }
 
-  def self.all_english_articles
+  scope :with_tags, -> (tags) { joins(:taggings).where('tag_id IN (?)', tags.map(&:id)) }
```

Comment:
Suggestion:

```scope :with_tags, -> (tags) { joins(:taggings).where('taggings.tag_id' => tags.map(&:id)) }```

This removes a little SQL from the ruby, and clarifies both to the reader and in the SQL that `tag_id` is on `taggings` — currently if someone went wild and added `tag_id` to the pages table this would break. No one will ever do that, but still.

Considered switching `tags.map(&:id)` to `tags.pluck('id')`, which would be smarter, but only works with ActiveRecord relations, so I figured it was a bad idea since you might want to pass in an array.